<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralConfigWidget</class>
 <widget class="QWidget" name="GeneralConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>313</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="confgeral_grid">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="11" column="1">
    <widget class="QRadioButton" name="landscape_rb">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Landscape</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="2" colspan="2">
    <widget class="QRadioButton" name="portrait_rb">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Portrait</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSpinBox" name="autosave_interv_spb">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="min_lbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>minute(s)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="autosave_interv_chk">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Autosave interval:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="print_grid_chk">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Print grid</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>181</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QSpinBox" name="grid_size_spb">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pixels_lbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>pixels</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="paper_lbl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Paper:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="orientation_lbl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Orientation:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="4">
    <widget class="QComboBox" name="paper_cmb">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>A0 (841 x 1189 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A1 (594 x 841 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A2 (420 x 594 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A3 (297 x 420 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A4 (210 x 297 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A5 (148 x 210 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A6 (105 x 148 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A7 (74 x 105 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A8 (52 x 74 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A9 (37 x 52 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B0 (1030 x 1456 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B1 (728 x 1030 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B10 (32 x 45 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B2 (515 x 728 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B3 (364 x 515 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B4 (257 x 364 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B5 (182 x 257 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B6 (128 x 182 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B7 (91 x 128 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B8 (64 x 91 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B9 (45 x 64 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C5E (163 x 229 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Comm10E (105 x 241 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DLE (110 x 220 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Executive (191 x 254 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Folio (210 x 330 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ledger (432 x 279 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Legal (216 x 356 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Letter (216 x 279 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tabloid (279 x 432 mm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom (Based on margins)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QComboBox" name="unity_cmb">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Milimeters</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pixels</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Inches</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Centimeter</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0" colspan="5">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QCheckBox" name="hide_ext_attribs_chk">
     <property name="text">
      <string>Hide table extended attributes</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2" rowspan="2" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="left_lbl">
       <property name="text">
        <string>Left:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="left_marg">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>23</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Left margin</string>
       </property>
       <property name="maximum">
        <double>5000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="top_lbl">
       <property name="text">
        <string>Top:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="top_marg">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>23</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Top margin</string>
       </property>
       <property name="maximum">
        <double>5000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="right_lbl">
       <property name="text">
        <string>Right:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="right_marg">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>23</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Right margin</string>
       </property>
       <property name="maximum">
        <double>5000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="bottom_lbl">
       <property name="text">
        <string>Bottom:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="bottom_marg">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>23</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Bottom margin</string>
       </property>
       <property name="maximum">
        <double>5000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="margins_lbl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Margins:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="4">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="oplist_size_lbl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Operation history:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSpinBox" name="oplist_size_spb">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>500</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="items_lbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>items</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="print_op_lbl">
     <property name="text">
      <string>Printing Options:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="3">
    <widget class="QCheckBox" name="print_pg_num_chk">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Print page numbers</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="hide_rel_name_chk">
     <property name="text">
      <string>Hide relationship name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="grid_size_lbl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Grid size:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="save_session_chk">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Save current session</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>grid_size_spb</tabstop>
  <tabstop>oplist_size_spb</tabstop>
  <tabstop>autosave_interv_chk</tabstop>
  <tabstop>autosave_interv_spb</tabstop>
  <tabstop>hide_rel_name_chk</tabstop>
  <tabstop>hide_ext_attribs_chk</tabstop>
  <tabstop>save_session_chk</tabstop>
  <tabstop>print_grid_chk</tabstop>
  <tabstop>print_pg_num_chk</tabstop>
  <tabstop>paper_cmb</tabstop>
  <tabstop>landscape_rb</tabstop>
  <tabstop>portrait_rb</tabstop>
  <tabstop>unity_cmb</tabstop>
  <tabstop>left_marg</tabstop>
  <tabstop>top_marg</tabstop>
  <tabstop>right_marg</tabstop>
  <tabstop>bottom_marg</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
